server/handlers/databaseHandler.js

```diff
@@ -155,9 +155,37 @@ let queryDatabaseBoolean = async (res, query, params, operationString) => {
   });
 };
 
+/**
```

Comment:
According to CodeClimate, the number of parameters nested inside of this method is bad practice. Though, I'm not sure how we can abstract much of this out at the moment.

Reply:
Yup—I feel that. We can come back to that in the future :)